### ISSUE TITLE
About the dataset concatenation

### ISSUE THREAD

**Issue by yuanhaoguo** [Yunfan-Li/Contrastive-Clustering#33]
Excellent work! We are grateful that the codes can be released for study.
I have a question about the creation of the dataset: 
dataset = data.ConcatDataset([train_dataset, test_dataset])
I guess here both train and test sets are used for training and testing. I can understand that in the task of unsupervised clustering, the true labels are invisible, so all data can be used for training. But I am wondering if this is a standard usage or definition in the field of "deep clustering", or I got a wrong understanding? Thanks~

**Comment by Yunfan-Li** [comment]
Most of the deep clustering methods concat the training and test set for both training and evaluation. Some works including ADC (Associative deep clustering: Training a classification network with no labels) and SCAN (SCAN: Learning to Classify Images without Labels) use training set for training and test set for evaluation. Personally, I feel that both settings could be adopted as long as you make it clear in the paper.
